README.md

```diff
@@ -215,6 +219,32 @@ script:
       bash <(curl -s https://codecov.io/bash) -f lcov.info;
 ```
 
+### grcov with GitLab CI and codecov.io
+
+```yaml
+codecov:
+  stage:                           coverage
+  variables:
+    CARGO_INCREMENTAL:             0
+    RUSTFLAGS:                     "-Zprofile -Zmir-opt-level=0 -Ccodegen-units=1
```

Comment:
`-Zmir-opt-level` is not in the list above, why is it useful? If it is, we should add it everywhere

Reply:
mostly for no-landing-pads. Tests have shown that with it, thee coverage is better.

Reply:
however, it requires nightly